models/event.json

```diff
@@ -296,124 +296,175 @@
+        },
+        "sameAs": {
+          "$ref": "./event-sameAs.json"
+        },
+        "workflowStatus": {
+          "$ref": "./event-workflowStatus.json"
+        },
+        "audience": {
+          "$ref": "./event-audience.json"
+        },
+        "languages": {
+          "$ref": "./event-languages.json"
+        },
+        "completedLanguages": {
+          "$ref": "./event-completedLanguages.json"
+        },
+        "typicalAgeRange": {
+          "$ref": "./event-typicalAgeRange.json"
+        },
+        "description": {
+          "$ref": "./event-description.json"
+        },
+        "priceInfo": {
+          "$ref": "./event-priceInfo.json"
+        },
+        "contactPoint": {
+          "$ref": "./event-contactPoint.json"
+        },
+        "bookingInfo": {
+          "$ref": "./event-bookingInfo.json"
+        },
+        "mediaObject": {
+          "$ref": "./event-mediaObject.json"
+        },
+        "image": {
+          "$ref": "./event-image.json"
+        },
+        "videos": {
+          "$ref": "./event-videos.json"
+        },
+        "labels": {
+          "$ref": "./event-labels.json"
+        },
+        "hiddenLabels": {
+          "$ref": "./event-hiddenLabels.json"
+        },
+        "production": {
+          "$ref": "./event-production.json"
+        },
+        "bookingAvailability": {
+          "$ref": "./event-bookingAvailability.json"
+        },
+        "created": {
+          "$ref": "./event-created.json"
+        },
+        "modified": {
+          "$ref": "./event-modified.json"
+        },
+        "creator": {
+          "$ref": "./event-creator.json"
+        }
+      },
+      "required": [
+        "@id",
+        "mainLanguage",
+        "name",
+        "terms",
+        "location",
+        "calendarType"
+      ]
     },
-    "hiddenLabels": {
-      "$ref": "./event-hiddenLabels.json"
+    {
+      "if": {
+        "required": [
+          "calendarType"
+        ],
+        "properties": {
+          "calendarType": {
+            "const": "periodic"
+          }
+        }
+      },
+      "then": {
+        "required": [
+          "startDate",
+          "endDate"
+        ]
+      }
     },
-    "production": {
-      "$ref": "./event-production.json"
+    {
+      "if": {
+        "required": [
+          "calendarType"
+        ],
+        "properties": {
+          "calendarType": {
+            "const": "single"
+          }
+        }
+      },
+      "then": {
+        "required": [
+          "startDate",
+          "endDate"
+        ]
+      }
     },
-    "bookingAvailability": {
-      "$ref": "./event-bookingAvailability.json"
+    {
+      "if": {
+        "required": [
+          "calendarType"
+        ],
+        "properties": {
+          "calendarType": {
+            "const": "multiple"
+          }
+        }
+      },
+      "then": {
+        "required": [
+          "startDate",
+          "endDate",
+          "subEvent"
+        ]
```

Comment:
For `multiple` only `subEvent` is required according to the existing calendar update schema https://github.com/cultuurnet/stoplight-docs-uitdatabank/blob/master/models/event-calendar-put.json#L47-L49